Enable caching of profiles.

This speeds up loading the (unchanged) profiles about 20 times.

Upstream doesn't enable caching because the cache directory is not
writeable at the time profiles are loaded in Ubuntu.

See also bnc#689458


Signed-off by: Christian Boltz <apparmor@cboltz.de>

Index: parser/parser.conf
===================================================================
--- parser/parser.conf_ORIG	2018-04-19 22:47:18.485179998 +0200
+++ parser/parser.conf	2018-04-19 22:51:12.084588654 +0200
@@ -31,7 +31,7 @@
 # match-string "pattern=aadfa audit perms=crwxamlk/ user::other"
 
 ## Turn creating/updating of the cache on by default
-#write-cache
+write-cache
 
 ## Show cache hits
 #show-cache

